edge-functions/vercel-og-nextjs/package.json

```diff
@@ -9,7 +9,7 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "@vercel/og": "0.0.14",
+    "@vercel/og": "0.0.20",
```

Comment:
we could go for `latest` like we have with React and Next.js

Reply:
Ack, wasn't sure which would be better here and opted for the safe side since it's not at v1 yet